### ISSUE TITLE
Find a way to displace comments better

### ISSUE THREAD

**Issue by patriciogonzalezvivo** [tangrams/tangram-play#15]

**Comment by louh** [comment]
@patriciogonzalezvivo Is this for comments in the editor itself?

**Comment by patriciogonzalezvivo** [comment]
yes, but not sure still how.... maybe by changing the background.

Also is not top priority

**Comment by louh** [comment]
@patriciogonzalezvivo This issue is a year old -- can you describe more about how comments need to be better?

**Comment by patriciogonzalezvivo** [comment]
If I remember better... is just a way to highlight comments in a nice way... the gray kind of fade them.
Also I was thinking if there is a way to make them more reactive, like:
- appearing when you hover... 
- parse them differently with markdown...

**Comment by louh** [comment]
Can you give me any examples or use cases where this is beneficial?
